app/classifier/components/Intervention/Intervention.spec.js

```diff
@@ -30,7 +30,14 @@ describe('Intervention', function () {
   });
   it('should show a notification message', () => {
     const newlineMsg = intervention.message + "\n"
-    expect(wrapper.find(Markdown).text()).to.equal(newlineMsg);
+    expect(wrapper.find(Markdown).first().text()).to.equal(newlineMsg);
+  });
+  it('should show an opt-out message', () => {
+    const expectedMarkdown = 'I do not want to take part in this messaging [study]' +
```

Comment:
By the way, is this split into multiple lines so that the linter doesn't complain? The concatenation is a bit strange.